python/meterpreter/tests/test_ext_server_stdapi.py

```diff
@@ -320,6 +320,27 @@ def test_stdapi_sys_config_getsid(self):
         ).get("value")
         self.assertRegex(sid, "S-1-5-.*")
 
+class ExtStdNetResolveTest(ExtServerStdApiTest):
+    def test_stdapi_net_resolve_host(self):
+        # Full request from msfconsole
+        request = b'\x00\x00\x00\x0c\x00\x02\x00\x01\x00\x00\x04\x00\x00\x00\x00)\x00\x01\x00\x0264769531726942037539492283558475\x00\x00\x00\x00\x13\x00\x01\x05xrapid7.com\x00\x00\x00\x00\x0c\x00\x02\x05\xa4\x00\x00\x00\x02'
+        response = bytes()
+        _result_code, result_tlvs = self.assertMethodErrorSuccess(
+            "stdapi_net_resolve_hosts", request, response
+        )
+
+        resolved_hosts = self.meterpreter_context["packet_get_tlv"](
+            result_tlvs, self.ext_server_stdapi["TLV_TYPE_RESOLVE_HOST_ENTRY"]
+        ).get("value")
+
+        resolved_hosts = self.meterpreter_context["packet_enum_tlvs"](
```

Comment:
Code smell: It looks like we're redefining this variable entirely, and it's no longer related to the previous definition - which makes it hard to reason about your code at a quick glance to identify issues 👀 

Solution: Don't redefine variables, or if you're forced to do so - make sure it's at least related to the same concept as the previous variable